api/v1beta2/types.go

```diff
@@ -1125,6 +1135,7 @@ type RemotePeeringConnection struct {
 // InternetGateway is used to specify the options for creating internet gateway.
 type InternetGateway struct {
 	// Skip specifies whether to skip creating internet gateway even if any one Subnet is public.
+	// In case of VCN being Skipped (Skip field of VCN set to true), this field should be true also
```

Comment:
NIT: these should say must 
```suggestion
	// In case of VCN being Skipped (Skip field of VCN set to true), this field must be true also
```